addons/website_event_track/static/src/js/service_worker.js

```diff
@@ -347,7 +354,7 @@ const processMessage = (data) => {
 };
 
 self.addEventListener("fetch", (event) => {
-    event.respondWith(processFetchRequest(event));
+    event.respondWith(processFetchRequest(event.request));
```

Comment:
I'm surprised this call works with your condition here above?

![image](https://user-images.githubusercontent.com/41282536/165748770-a1a00c92-1c0f-46ad-9775-d96d1a16deb2.png)